src/engraving/libmscore/score.h

```diff
@@ -785,7 +785,7 @@ class Score : public EngravingObject
 
     void changeSelectedNotesVoice(voice_idx_t);
 
-    const std::vector<Part*>& parts() const { return _parts; }
+    const std::vector<Part*>& parts() const;
```

Comment:
There was a very strange crash if we use this code after compiling its clang and cpp20